src/main/resources/templates/home.html

```diff
@@ -11,11 +11,13 @@
 <div class="success-message" th:if="${session.loginSuccessful}" th:text="${session.loginSuccessful}"></div>
 <header class="header">
     <nav class="nav-bar">
-        <a href="home" class="logo"><i class='bx bx-dish'></i></a>
-        <a href="home">Home</a>
-        <a href="#">About</a>
-        <a href="#">Contact</a>
-        <a href="#">Help</a>
+
+        <a href="home?page=home" class="logo"><i class='bx bx-dish'></i></a>
+        <a href="home?page=home">Home</a>
```

Comment:
I changed it so that the page you are on displays in the URL when you navigate to that page.

https://developers.google.com/search/docs/crawling-indexing/url-structure